jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/basic/BasicFiltersViewImpl.java

```diff
@@ -151,15 +151,20 @@ public HTMLElement getElement() {
     public void addTextFilter(final String label,
                               final String placeholder,
                               final Consumer<ActiveFilterItem<String>> callback) {
+        addTextFilter(label, placeholder, false, callback);
+    }
 
+    @Override
+    public void addTextFilter(String label, String placeholder, boolean disableFiltersInputHelp, Consumer<ActiveFilterItem<String>> callback) {
+        filtersInputHelp.setDisabled(disableFiltersInputHelp);
```

Comment:
@bxf12315 even the filtersInputHelp is set to disabled at firefox the popover is shown over the grey help button

Reply:
@bxf12315 I believe this still needs attention

Reply:
also, has the issue with using _*_ been fixed?